src/serializer/MapFishPrintV3GeoJsonSerializer.js

```diff
@@ -374,12 +377,14 @@ export class MapFishPrintV3GeoJsonSerializer extends BaseSerializer {
    *                  instance.
    */
   writeFillStyle = olFillStyle => {
-    if (!(olFillStyle instanceof OlStyleFill)) {
+    if (olFillStyle && !(olFillStyle instanceof OlStyleFill)) {
       return {};
     }
 
     return {
-      color: olFillStyle.getColor()
+      // if no fill is given set it to full transparent. In some elemnts with stroke and no fill, the object cannot
+      // be rendered for some reason
+      color: olFillStyle ? olFillStyle.getColor() : 'rgb(0,0,0,0)'
```

Comment:
Not sure, but I would expect rgb**a**(0,0,0,0) here.

Reply:
I replaced it although both seem to work the same way. In any case, rgba makes more sense